mycroft/util/__init__.py

```diff
@@ -109,5 +111,21 @@ def connected(host="8.8.8.8", port=53, timeout=3):
             return False
 
 
+def create_signal(signalName):
```

Comment:
Sorry for the 9-months late review. I have concerns about this on picroft (and mark 1s) as it will unnecessarily abuse the file system. SD cards have a finite number of writes, so I had been purposely avoiding things like this. 

A better practice would probably be something like emitter.once , but requires that you put more thought into an order of operations of sorts (i.e. I must set my listener before the event occurs, likely by setting the listener and then beginning a chain of events). An example from the pyeventemitter docs:

```
import event_emitter as events

em = events.EventEmitter()

def hello(who):
    print('Hello {}'.format(who))

em.once('hello', hello)
em.emit('hello', who='World')  # prints Hello World
em.emit('hello', who='World')  # nothing happens
```